Oqtane.Client/Modules/Controls/Label.razor

```diff
@@ -43,16 +43,12 @@ else
 
         _openLabel += ">";
 
-        if (IsLocalizable)
+        var text = Localize("Text", String.Empty);
```

Comment:
It's working @sbwalker but there's an extra work here coz I'm unable to read the `ChildContent`, I'm asking why it's not string property?!!

Reply:
I think I was unable to figure out how to utilize @ChildContent within a tag:

```
<label for="name" class="class">@ChildContent</label> 
```
The razor compiler could not handle it so I had to use an ugly hack. If you are able to create a better implementation which has the same interface and produces the same output result, please go ahead.

Reply:
I suffered when I worked with the render tree ;) so it's fine for now, all the cases are handled